docs/aws/aws.md

```diff
@@ -196,3 +196,24 @@ See the egress golang examples above, and replace the line starting with `out :=
 ```go
 out := cli.VerifyDns(context.TODO(), "vpcID")
 ```
+
+## Interface ##
+
+### Platform ###
+The platform interface is used to inform network-verifier of the platform type it is running on. It is used to represent specific Platform types and how they map to their respective platform. It contains an array of length containing valid names for each platform type.
+```
+type Platform struct {
+	// names holds 3 unique lowercase names of the Platform (e.g., "aws"). We use a fixed-
+	// size array so that this struct remains comparable. Any of the 3 values can be used to refer
+	// to this specific Platform via Platform.ByName(), but only the first (element
+	// 0) element will be the "preferred name" returned by Platform.String()
+	names [3]string
+}
+```
+
+Currently network-verifier supports three implementations for AWS.
+- AWSClassic
+- AWSHCP
+- AWSHCPZeroEgress
```

Comment:
Along with the above advice of consolidating, maybe a table for each `AWS` and `GCP` with the implementation names listed?

Reply:
I think this would make sense if we had more implementations for each, but since GCP only has the one and AWS three I think the bulleted list is okay.  I'm still open to changing it

Reply:
Yeah no strong opinion here, list seems good